tests/serverpod_test_server/test/generated_methods/copy_with_mutable_test.dart

```diff
@@ -235,5 +235,110 @@ void main() {
       expect(mapDataCopy.data['b']?.num, 2);
       expect(mapDataCopy.data['b']?.id, null);
     });
+
+    group('Given an object with nested Lists and Maps when calling copyWith',
```

Comment:
Question: Are we really validating the mutability here? Are we not validating that the values are still accessible? Maybe split mutability and accessibility validation.

Reply:
Nice spot! I missed the top level `group`. Do you often have a top level group like that or is it older code that doesn't strictly adhere to given/when/then?